### ISSUE TITLE
Random configuration issues

### ISSUE THREAD

**Issue by f0rkz** [f0rkz/gameserver-daemon#3]
```
Traceback (most recent call last):
  File "gameserver-daemon.py", line 141, in <module>
    if os.path.isfile(CONFIG_FILE) and args.steamcmd:
```

**Comment by f0rkz** [comment]
```
Traceback (most recent call last):
  File "gameserver-daemon.py", line 141, in <module>
    if os.path.isfile(CONFIG_FILE) and args.steamcmd:
AttributeError: 'Namespace' object has no attribute 'steamcmd'
```

**Comment by f0rkz** [comment]
```
Success! App '4020' fully installed.
Traceback (most recent call last):
  File "gameserver-daemon.py", line 160, in <module>
    if os.path.isfile(CONFIG_FILE) and engine == 'srcds' and args.motd:
AttributeError: 'Namespace' object has no attribute 'motd'
```

**Comment by f0rkz** [comment]
Finally fixed in d5efdb0 !
